.circleci/config.yml

```diff
@@ -338,6 +330,31 @@ jobs:
          name: "Run Example Binaries"
          command: |
            find _build/debug/velox/examples/ -maxdepth 1 -type f -executable -exec "{}" \;
+      - run:
```

Comment:
nice! At some point it would be nice to centralize this and more tests into a "backwards compatibility check" sort of CI pipeline